keel-sql/keel-sql.gradle.kts

```diff
@@ -48,7 +48,7 @@ dependencies {
   implementation("com.netflix.spinnaker.kork:kork-sql")
   implementation("org.springframework:spring-jdbc")
   implementation("org.springframework:spring-tx")
-  implementation("org.jooq:jooq:3.11.11")
+  implementation("org.jooq:jooq:3.13.2")
```

Comment:
I don't know why we were pinning to a particular version before. I added it now until we can get 3.13.x in Kork.